### PR TITLE
[RNMobile] Set tintColor for ActionSheetIOS

### PR DESCRIPTION
**Pull request by SiobhyB**
## Related PRs

* `gutenberg-mobile`: https://github.com/wordpress-mobile/gutenberg-mobile/pull/5074

## What?

This PR explicitly sets a `tintColor` on the [ActionSheetIOS](https://reactnative.dev/docs/actionsheetios) component.

## Why?

At the moment, the `tintColor` is not explicitly defined and it's easy for it be unintentionally overridden by host apps. Explicitly setting the value will avoid any unintentional overides.

## How?

The [ActionSheetIOS](https://reactnative.dev/docs/actionsheetios) component's `tintColor` prop is used in combination with the `usePreferredColorScheme` hook to set different colours for light/dark mode. 

## Testing Instructions

1. Navigate to the editor in the iOS WordPress app.
2. To access the ActionSheetIOS component, add an image block and tap the option to `add image`. Another option for accessing the component is to tap the ellipsis in the bottom right of a block for the actions menu.
3. Confirm that this PR doesn't introduce any unexpected colour changes in the WordPress app.

## Screenshots or screencast 

| Light  | Dark |
| ------------- | ------------- |
|  |  |

### CODE REVIEW

**Comment by SiobhyB** [comment]
I've created a separate PR to update all the CSS selectors in the picker's files to follow BEM naming convention at https://github.com/WordPress/gutenberg/pull/43036. (I didn't want to send this PR off-course by doing it directly here, but also didn't want to leave the component's `scss` file with only some selectors using the convention.)

File: packages/components/src/mobile/picker/styles.native.scss [hunk 1]
```diff
@@ -19,3 +19,11 @@
...
 	padding-right: 0;
 }
+
+.components-picker__button-title {
+	color: $blue-50;
+}
+
+.components-picker__button-title--dark {
+	color: $blue-30;
+}
```
Comment (dcalhoun) [inline packages/components/src/mobile/picker/styles.native.scss:29]: These static colors make sense to me for the current goal — a consistent color within the editor. Long term, I would imagine we should revert this change so that the host app once again can control the picker colors.

**Review by dcalhoun — APPROVED** [review]
🚀 LGTM. I tested these changes on an iPhone SE (iOS 15.6). Thanks!
